decidim-core/app/views/decidim/doorkeeper/authorizations/new.html.erb

```diff
@@ -13,16 +13,41 @@
 
     <p><%= t("this_application_will_be_able_to", scope: "decidim.doorkeeper.authorizations.new") %></p>
     <ul class="list-disc ml-4">
-      <li><%= t("see_name", scope: "decidim.doorkeeper.authorizations.new") %></li>
-      <li><%= t("see_username", scope: "decidim.doorkeeper.authorizations.new") %></li>
-      <li><%= t("see_email", scope: "decidim.doorkeeper.authorizations.new") %></li>
+      <% if @scopes.include?("profile") || @scopes.include?("user") %>
+        <li><%= t("see_name", scope: "decidim.doorkeeper.authorizations.new") %></li>
+        <li><%= t("see_username", scope: "decidim.doorkeeper.authorizations.new") %></li>
+        <li><%= t("see_email", scope: "decidim.doorkeeper.authorizations.new") %></li>
+      <% end %>
+      <% if @scopes.include?("user") %>
+        <li><%= t("update_profile", scope: "decidim.doorkeeper.authorizations.new") %></li>
+      <% end %>
+      <% if @scopes.include?("api:read") %>
+        <li><%= t("api_read", scope: "decidim.doorkeeper.authorizations.new") %></li>
+      <% end %>
+      <% if @scopes.include?("api:write") %>
+        <li><%= t("publish_content", scope: "decidim.doorkeeper.authorizations.new") %></li>
+      <% end %>
```

Comment:
This would make harder to register new scopes. I am thinking that each module (space or component) would add a: 

```
  component.api_scope = %w(blogs:read blogs:write)
```

Yet, some installations may have installed blogs, while others would not.